backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/manufacturing/workflows_api/activity_handlers/issue/json/JsonRawMaterialsIssueLineStep.java

```diff
@@ -39,7 +44,11 @@ public static JsonRawMaterialsIssueLineStep of(RawMaterialsIssueStep step, JsonO
 				.productId(String.valueOf(step.getProductId().getRepoId()))
 				.productName(step.getProductName().translate(jsonOpts.getAdLanguage()))
 				.locatorName(step.getIssueFromLocator().getCaption())
-				.huQRCode(step.getIssueFromHU().getBarcode().toRenderedJson())
+				.locatorQrCode(step.getIssueFromLocator().getQrCode().toGlobalQRCodeJsonString())
+				.huQRCode(Optional.ofNullable(step.getIssueFromHU().getBarcode())
```

Comment:
for future: next time when u feel the need of of using Optional.ofNullable(..)....orElse.... , maybe it would be better to change the getBarcode method and return Optional